draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1236,6 +1236,11 @@ If the certificate is logged, an SCT MUST be issued. Logging the certificate is
 useful, because monitors ({{monitor}}) can then detect these encoding errors,
 which may be accepted by some TLS clients.
 
+If `submission` is an accepted trust anchor whose certifier is neither an
+accepted trust anchor nor the first element of `chain`, then the log MUST return
```

Comment:
I don't think so.  It says "whose certifier".  A non-self-signed accepted trust anchor is not its own certifier, by definition.